### DISCUSSION TITLE
Mitsubishi Starmax

### DISCUSSION THREAD

**Discussion by J-S-LIM** [echavet/MitsubishiCN105ESPHome#174] [Q&A]
Hi,

Chanced upon this thread as I embark onto Home Assistant.
Am thinking use the ESP32 to integrate Mitsubishi into Home Assistant.

Can I ask if anyone tried it on the below Mitsubishi System:
Outdoor: MXY-2H20, MXY-3H28
Indoor: MSXY-FP10, MSXY-FP13, MSXY-FP18

https://www.mitsubishielectric.com.sg/getattachment/b0b0633d-3972-4394-8237-c6bfa06f92bf/Mitsubishi-Electric-Starmex-R32-Catalogue.pdf

A third party company sell the Adapter at a costly price. However, my main purpose is to integrate into Home Assistant.
https://coolserve.com/product/vida-mitsubishi-electric-wifi-adapter/

Country: Singapore.

Thank you for your time.

**Comment by iblackwood** [comment]
You need the CN105 connector inside the indoor unit.
I could not see any reference to these units having the WiFi module as an option from Mitsubishi. Even that adapter you link to needs the CN105 connector in the indoor unit to plug into.
If you can find out whether the Indoor units support the Mitsubishi MAC-568IF-E Wi-Fi Controller then you should know.
The Mitsubishi website for Australia says the wifi controller supports "M Series" but I would not take that as given for Singapore models too !

Sorry, haven't tried it, but if you have the connector it will likely work for at least most functions.

**Comment by iphoting** [comment]
Somebody reported success: https://github.com/echavet/MitsubishiCN105ESPHome/discussions/606
